torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -741,6 +741,22 @@ at::Tensor& XLANativeFunctions::_index_put_impl_(
                                                    accumulate);
 }
 
+std::tuple<at::Tensor, at::Tensor> XLANativeFunctions::_linalg_eigh(
+    const at::Tensor& self, c10::string_view uplo, bool compute_v) {
+  TORCH_LAZY_FN_COUNTER_TIMED_TRACING("xla::");
+  if (!compute_v) {
+    // Fallback to aten in case of `eigvalsh`, which does not compute
+    // eigenvectors but requires numerically stable gradients.
+    return at::native::call_fallback_fn<&xla_fallback,
```

Comment:
I understand we only need to lower torch.linalg.eigh. But in case we need to lower eigvalsh later, would `requires numerically stable gradients` be a blocker so we have to fall back to aten?

Reply:
So I read the PyTorch docs again and I think I misunderstood it initially. What PyTorch doc suggests is that the gradients of the _eigenvectors_ are unstable. Therefore, if the user calls `eigvalsh`, they will only get _eigenvalues_ and thus the gradients will be stable. I removed this misleading comment.

If we want to support `eigvalsh` later the simplest way is probably discarding the eigenvectors from XLA and also figuring out what to return for the second `at::Tensor` tuple member.